client/dashboard/utils/site-provider.ts

```diff
@@ -1,16 +1,35 @@
 import type { Site } from '@automattic/api-core';
 
-export const DEFAULT_PROVIDER_NAME = 'WordPress.com';
+export const DEFAULT_PROVIDER_NAME = 'Unknown';
+
+const isWordPressComProvider = ( site: Site ) => {
+	return (
+		site.is_wpcom_atomic ||
+		site.is_a8c ||
+		site.is_garden ||
```

Comment:
Nit: It makes more sense to me to depend on these utility functions instead of raw data https://github.com/Automattic/wp-calypso/blob/trunk/client/dashboard/utils/site-types.ts 

My thinking is that since they already exist, they should be the "source of truth" to determine the site type.